docs/security.md

```diff
@@ -214,7 +214,7 @@ The following table describes the different options available for configuring th
     Cipher mode to use. Defaults "AES/CTR/NoPadding" for backward compatibility, which is not authenticated. 
     Recommended to use "AES/GCM/NoPadding", which is an authenticated encryption mode.
   </td>
-  <td>4.0.0</td>
+  <td>4.0.0, 3.5.2, 3.4.4</td>
```

Comment:
We will set this field to '3.4.4' as usual. It's okay when we fix it in one or two branches, but issues arise when we continue to backport more than 2 branches. The problem here is that it only tells us that 3.4.x (x>=4) and x.y.z (x>=4), but it can't tell us which version of 3.5 has it.

I suggest that we add the full list of fixed versions according to the corresponding JIRA ticket.

See the discussion here https://github.com/apache/spark/pull/47325#discussion_r1676826574

Reply:
> We will set this field to '3.4.4' as usual. It's okay when we fix it in one or two branches, but issues arise when we continue to backport more than 2 branches. The problem here is that it only tells us that 3.4.x (x>=4) and x.y.z (x>=4), but it can't tell us which version of 3.5 has it.
> 
> I suggest that we add the full list of fixed versions according to the corresponding JIRA ticket.
> 
> See the discussion here [#47325 (comment)](https://github.com/apache/spark/pull/47325#discussion_r1676826574)

To @yaooqinn , actually, I disagreed with the suggestion.